default/scripting/ship_hulls/monster/monster.macros

```diff
@@ -17,6 +17,10 @@ MONSTER_MOVE_ALWAYS
                 Turn low = 10
                 Stationary
                 Unowned
+                Or [
```

Comment:
this should be an And i think

Reply:
Double checked this in game again with both set to on and off, it's correct as is.

It's:  If (70% or (can stop = false)) then move

Your suggested change would be: If (30% and (can stop=true)) then move. This would prevent any movement when can stop = false

Reply:
Yep, tested your suggested change and it does result in:

1. If can stop = false then never move
2. If can stop = true then 30% chance of move (instead of 70% chance of movement)